corehq/apps/users/management/commands/delete_web_user.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from getpass import getpass
+
+from django.contrib.auth import authenticate
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models import WebUser
+
+
+class Command(BaseCommand):
+    help = "Deletes the given web user"
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+
+    def handle(self, username, **options):
+        print("Authenticate:")
+        user_username = input("Please enter your username: ")
+        user_password = getpass()
```

Comment:
Is this necessary?  Whoever is running this has privileges to run management commands, so I'd presume they're trusted.  Do you know if calling authenticate like this will work for users with 2FA enabled?

Reply:
its a nice way of ensuring that the person who runs this, puts their own username and not anything random or someone else's and then its incorrectly audited for someone else.

2FA won't be asked for or checked here.